docs/examples/warming-stripe.json

```diff
@@ -0,0 +1,264 @@
+{
+  "$schema": "https://vega.github.io/schema/vega/v5.json",
+  "description": "Warming Stripes Chart. Originally designed by Ed Hawkins.",
+  "usermeta": {
```

Comment:
This is not something we have added to examples in the vega gallery. Maybe remove?

Reply:
ok